src/reynir_correct/errtokenizer.py

```diff
@@ -2800,7 +2831,15 @@ def check_taboo_words(token_stream: Iterable[CorrectToken]) -> Iterator[CorrectT
 
     for token in token_stream:
         # Check taboo words
-        if token.has_meanings:
+        if (  # type: ignore
+            token.val is not None
+            and token.has_meanings
+            and token.txt not in NOT_TABOO
+            and isinstance(token.val, list)  # List of BIN_Tuple
+            and not any(v.stofn + "_" + v.ordfl in NOT_TABOO for v in token.val)  # type: ignore
```

Comment:
OK, here is an accurate comparison, but this requires that an underscore is present in NOT_TABOO. So the two NOT_TABOO checks are incompatible.

Reply:
As explained above, NOT_TABOO contains both single word forms, that are checked with "token.txt not in NOT_TABOO", and then lemmas+PoS, checked with "not any...".
The following shows output for the current implementation:
`Ég er nýbúinn að hoppa á hommann út af nýbúanum.
Ég er nýbúinn að hoppa á hommann út af innflytjandi.
009-009: T001/w Óheppilegt eða óviðurkvæmilegt orð, skárra væri t.d. 'innflytjandi' | 'nýbúanum' -> 'innflytjandi' | None
`